### ISSUE TITLE
Authentication of Cody extension in VS Codium fails because of hard-coded vscode:// link.

### ISSUE THREAD

**Issue by LightTemplar** [sourcegraph/sourcegraph#55083]
- **Sourcegraph version:** "Sourcegraph.com" 
- **Platform information:** POP!_OS 22.04, Firefox: 115.0 (64-bit)
- VS Codium:
```
Version: 1.80.0
Release: 23188
Commit: 9d3b0fa210fb4535589e8296483a5b43fe5e660a
Date: 2023-07-07T18:21:29.357Z
Electron: 22.3.14
ElectronBuildId: undefined
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.2.6-76060206-generic
```
#### Description:
When authorizing the Cody VS Code extension in VS Codium, the authentication flow does not work properly.

#### Steps to reproduce:

1. Install Cody extension in VS Codium
1. Initiate authentication flow
1. After authenticating in browser, the redirect tries to open a vscode:// link in VS Codium

#### Expected behavior:
The authorization link should open within VS Codium to complete authorization.

#### Actual behavior:
The vscode:// link fails to open in VS Codium, leaving authentication incomplete.

#### Root cause:
The authentication flow seems to use hard-coded vscode:// links instead of vscodium:// links. VS Codium requires vscodium:// protocol to handle custom internal links.

#### Proposed solution:
Detect if editor is VS Codium and use vscodium:// links when needed.

**Comment by thednp** [comment]
Has anyone managed to get cody to work with VS Codium?

**Comment by wyrd-code** [comment]
Yes, there is an issue with instructions that I found but can't find again so I'll retype here.

Create a json file on disk with your access token structured like this:

```json
{
  "token": "YOUR_TOKEN_HERE"
}
``` 
And add this line to VSCodium json settings:

```json
  "cody.experimental.localTokenPath": "TOKEN_FILE_PATH_HERE",
```
Then  restart VSCodium and it should work.

**Comment by thednp** [comment]
@wyrd-code thank you!

**Comment by AoAnima** [comment]
> Yes, there is an issue with instructions that I found but can't find again so I'll retype here.
> 
> Create a json file on disk with your access token structured like this:
> 
> ```json
> {
>   "token": "YOUR_TOKEN_HERE"
> }
> ```
> 
> And add this line to VSCodium json settings:
> 
> ```json
>   "cody.experimental.localTokenPath": "TOKEN_FILE_PATH_HERE",
> ```
> 
> Then restart VSCodium and it should work.

Not work in cody 1.2.1706540836
